gzdoom@4.11.3: download via aria2 failed, github-actions bot mistakenly thinks the files can be downloaded just fine

```shell
PS C:\Users\[Redacted]> scoop update gzdoom
gzdoom: 4.11.2 -> 4.11.3
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Updating one outdated app:
Updating 'gzdoom' (4.11.2 -> 4.11.3)
Downloading new version
Starting download with aria2 ...
Download: 10/30 17:56:52 [ERROR] CUID#8 - Download aborted. URI=https://github.com/coelckers/gzdoom/releases/download/g4.11.3/gzDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/ZDoom/gzdoom/releases/download/g4.11.3/gzdoom-4Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 2b7527|ERR |       0B/s|C:/Users/[Redacted]/scoop/cache/gzdoom#4.11.3#https_github.com_coelckers_gzdoom_releases_download_g4.11.3_gzdoom-4-11-3-Windows-64bit.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/coelckers/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3-Windows-64bit.zip
    referer=https://github.com/coelckers/gzdoom/releases/download/g4.11.3/
    dir=C:\Users\[Redacted]\scoop\cache
    out=gzdoom#4.11.3#https_github.com_coelckers_gzdoom_releases_download_g4.11.3_gzdoom-4-11-3-Windows-64bit.zip

ERROR & 'C:\Users\[Redacted]\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\[Redacted]\scoop\cache\gzdoom.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=51832 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/Calinou/scoop-games/issues/new?title=gzdoom%404.11.3%3a+download+via+aria2+failed
```

In short, the problem is this link `https://github.com/coelckers/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3-Windows-64bit.zip` is a link that goes to a Not Found error screen... the link that needs to be in there is [this one](https://github.com/ZDoom/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3.a.-Windows-64bit.zip): `https://github.com/ZDoom/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3.a.-Windows-64bit.zip`

--- a/bucket/gzdoom.json
+++ b/bucket/gzdoom.json
@@ -10,8 +10,8 @@
     ],
     "architecture": {
         "64bit": {
-            "url": "https://github.com/coelckers/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3-Windows-64bit.zip",
-            "hash": "64b7248e6638fe7699135812cb4a7b7d7b9cd818cc6ad6ae677e39f4aef795b4"
+            "url": "https://github.com/ZDoom/gzdoom/releases/download/g4.11.3/gzdoom-4-11-3.a.-Windows-64bit.zip",
+            "hash": "9ba87f54b7da13d65ef295404480441f727479b44efb65a51dae54691e2786b7"
         }
     },
     "pre_install": [
@@ -34,13 +34,13 @@
         "Screenshots"
     ],
     "checkver": {
-        "github": "https://github.com/coelckers/gzdoom",
+        "github": "https://github.com/ZDoom/gzdoom",
         "regex": "/releases/tag/(?:g)?([\\w.]+)"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/coelckers/gzdoom/releases/download/g$version/gzdoom-$dashVersion-Windows-64bit.zip"
+                "url": "https://github.com/ZDoom/gzdoom/releases/download/g$version/gzdoom-$dashVersion-Windows-64bit.zip"
             }
         }
     }

